medallion/common.py

```diff
@@ -3,9 +3,12 @@
 import threading
 import uuid
 
+from flask import Flask
 import pytz
 from six import iteritems
 
+APPLICATION_INSTANCE = Flask("taxii-server")
```

Comment:
I will change it to medallion, but before it was defined in the __init__.py file using this:

> application_instance = Flask(__name__)

Since it moved to common.py, it would be medallion.common, so I will hardcode it as "medallion"